auth/store.go

```diff
@@ -150,6 +150,9 @@ type AuthStore interface {
 
 	// CheckPassword checks a given pair of username and password is correct
 	CheckPassword(username, password string) (uint64, error)
+
+	// IsAuthEnabled checks authentication is enabled or not
+	IsAuthEnabled() bool
```

Comment:
Ah, my writing was bad. Of course I didn't intend to remove the function, just only make it local again. I'll add conditions based on authStore is nil or not to etcdserver side.